config/hubs/openscapes.cluster.yaml

```diff
@@ -120,21 +139,6 @@ hubs:
               replicas: 0
             userScheduler:
               enabled: false
-          proxy:
-            service:
-              type: LoadBalancer
-            https:
-              enabled: true
-            chp:
-              nodeSelector: {}
-              tolerations:
-                - key: "node-role.kubernetes.io/master"
-                  effect: "NoSchedule"
-            traefik:
-              nodeSelector: {}
-              tolerations:
-                - key: "node-role.kubernetes.io/master"
-                  effect: "NoSchedule"
```

Comment:
On line 151 (in the `hub`section below) there are still some tolerations that you may want to remove as well?